recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.7.1" %}
+{% set md5 = "c47b36ceb17eaff3ecfab3bcd347d0df" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Can we break this out into `build.sh` and `bld.bat`?

Once we do this, we will need to delete the entrypoints in both. There is an environment variable `PY_VER`, which can be used to help.